docs/source/conf.py

```diff
@@ -13,31 +13,32 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath('...'))
+sys.path.insert(0, os.path.abspath("..."))
```

Comment:
```suggestion
```

Since this file is being touched in this PR, can we update it to reflect that changes in the commit below?

See the commit message for additional details.


- https://github.com/rapidsai/cuspatial/pull/868/commits/1ca27370930687575f37f83d634738db59913d07

The `sys.path`, `import`, and commented lines can all be removed.

We'll just want to add a copyright header back after removing the other commented lines.

Reply:
thanks, I did not realize these were never used, good catch. Although, we do require os package to execute `os.environ.get`

Reply:
resolved it by removing block using `os` since it's not being used.